src/main/java/core/basesyntax/model/FruitDto.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class FruitDto {
+    private String operation;
+    private String fruit;
+    private int quantity;
+    private LocalDate expirationDate;
```

Comment:
with buy operation it's not expirationDate it's purchase date

Reply:
changed